### PR TITLE
chore(curriculum): adding content to DOM review page 

### PR DESCRIPTION
**Pull request by jdwilkin4**
Checklist:

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/how-to-open-a-pull-request/).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

### CODE REVIEW

**Review by moT01 — APPROVED** [review]

File: curriculum/challenges/english/25-front-end-development/review-dom-manipulation-and-click-events-with-javascript/6723cc7a8e7aa3b9befd4bac.md [hunk 1]
```diff
@@ -9,7 +9,347 @@ dashedName: review-dom-manipulation-and-click-events-with-javascript
 
 Review the concepts below to prepare for the upcoming quiz.
 
+## Working with the DOM and Web API's
 
+- **API**: An API(Application Programming Interface) is a set of rules and protocols that allow software applications to communicate with each other and exchange data efficiently.
```
Comment (Sembauke) [inline curriculum/challenges/english/25-front-end-development/review-dom-manipulation-and-click-events-with-javascript/6723cc7a8e7aa3b9befd4bac.md:14]:
```suggestion
- **API**: An API (Application Programming Interface) is a set of rules and protocols that allow software applications to communicate with each other and exchange data efficiently. 
```

File: curriculum/challenges/english/25-front-end-development/review-dom-manipulation-and-click-events-with-javascript/6723cc7a8e7aa3b9befd4bac.md [hunk 2]
```diff
@@ -9,7 +9,347 @@ dashedName: review-dom-manipulation-and-click-events-with-javascript
 
 Review the concepts below to prepare for the upcoming quiz.
 
+## Working with the DOM and Web API's
 
+- **API**: An API(Application Programming Interface) is a set of rules and protocols that allow software applications to communicate with each other and exchange data efficiently. 
+- **Web API**: Web APIs are specifically designed for web applications. These types of API’s are often divided into two main categories: browser APIs and third-party APIs.
```
Comment (Sembauke) [inline curriculum/challenges/english/25-front-end-development/review-dom-manipulation-and-click-events-with-javascript/6723cc7a8e7aa3b9befd4bac.md:15]:
```suggestion
- **Web API**: Web APIs are specifically designed for web applications. These types of APIs are often divided into two main categories: browser APIs and third-party APIs.
```

File: curriculum/challenges/english/25-front-end-development/review-dom-manipulation-and-click-events-with-javascript/6723cc7a8e7aa3b9befd4bac.md [hunk 3]
```diff
@@ -9,7 +9,347 @@ dashedName: review-dom-manipulation-and-click-events-with-javascript
...
+- **Browser APIs**: These APIs expose data from the browser. As a web developer, you can access and manipulate this data using JavaScript.
+- **Third-Party APIs**: These are not builtin into the browser by default. You have to retrieve their code in some way. Usually, they will have detailed documentation explaining how to use their services. An example is the Google Maps API, which you can use to display interactive maps on your website.
```
Comment (Sembauke) [inline curriculum/challenges/english/25-front-end-development/review-dom-manipulation-and-click-events-with-javascript/6723cc7a8e7aa3b9befd4bac.md:17]:
```suggestion
- **Third-Party APIs**: These are not built into the browser by default. You have to retrieve their code in some way. Usually, they will have detailed documentation explaining how to use their services. An example is the Google Maps API, which you can use to display interactive maps on your website.
```

**Review by Sembauke — COMMENTED** [review]
Nit picks and one error.

**Review by naomi-lgbt — APPROVED** [review]

**Pull request merged**